modules/installation-aws-marketplace-subscribe.adoc

```diff
@@ -0,0 +1,64 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_aws/installing-aws-customizations.adoc
+// * installing/installing_aws/installing-aws-user-infra.adoc
```

Comment:
Looks like it's missing in GovCloud section: http://file.rdu.redhat.com/mpytlak/osdocs-3030/installing/installing_aws/installing-aws-government-region.html

Reply:
Thank you. Nice catch. Added to GovCloud topic.